test/utils/describeConformanceV5.js

```diff
@@ -252,6 +275,10 @@ export default function describeConformanceV5(minimalElement, getOptions) {
   const { after: runAfterHook = () => {}, only = Object.keys(fullSuite), skip = [] } = getOptions();
 
   describe('Material-UI component API', () => {
+    before(() => {
+      // this would help catch missing properties and fixing tests faster
+      validateOptions(getOptions(), skip);
```

Comment:
Added this checker after facing this kind of error a lot of time 😂.
The root cause is that I missed some properties when doing migration in the test but the error does not lead to the root cause.

This is the error before the validator is added.
![image](https://user-images.githubusercontent.com/18292247/115180326-87c6f600-a0ff-11eb-8d31-e303555fc4c7.png)

This is after.
![image](https://user-images.githubusercontent.com/18292247/115180367-9b725c80-a0ff-11eb-8b9f-d2e59c2ce27d.png)

I think this will help DX when writing tests and does not need to spend 5 mins figure out what's wrong.

Reply:
Awesome!

Reply:
Should we extract this in a separate PR?

Reply:
I don't see much value to make another PR for this, it is kinda internal. @oliviertassinari what do you think?

Reply:
I'd say we should always extract as separate PRs not related to the title of the current PR, it's easier to search for changes later that way (ex search with label `test`). No hard preference as it is only test change, but whatever, I think it's good practice in general.

Reply:
got it, will separate it out.